packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -516,8 +518,8 @@ export class ProjectLifecycleManager {
       d.currentTestingType = testingType
       d.wizard.chosenBundler = null
       d.wizard.chosenFramework = null
-      if (d.currentProjectData) {
-        d.currentProjectData.testingTypeData = makeTestingTypeData(testingType)
+      if (testingType) {
```

Comment:
By resetting this error, are we potentially swallowing errors from global state and/or the initial project selected view int eh launchpad? Or are these already emitted? Not sure if we have any existing tests around the error propagation?

Reply:
Tests for this area:
- warnings: https://github.com/cypress-io/cypress/blob/develop/packages/launchpad/cypress/e2e/config-warning.cy.ts
errors: https://github.com/cypress-io/cypress/blob/develop/packages/launchpad/cypress/e2e/error-handling.cy.ts
I tried deleting this entire part and **not** clearing the error, and actually, everything still passes. Maybe we just remove this entirely.

If there is ever an **error** we have a pattern - show this overlay:

![image](https://user-images.githubusercontent.com/19196536/182511705-3617f2d7-0a88-4708-ad14-6fb2b1d5e304.png)

The only way out is "Try again" after you fix the error, or to close Cypress. Whether this is the ideal behavior is up for debate, but it suggests that it's fine to clobber this error.

The only time we seem to show errors is things relating to project config. I can't think of any situations where we show an error in **global** mode (eg, where you haven't chosen a project). I tried to create one, but no luck. Open mode tests are here: https://github.com/cypress-io/cypress/blob/develop/packages/launchpad/cypress/e2e/open-mode.cy.ts (none for errors, perhaps an error state in global mode is just not a possible state?)